support-frontend/build.sbt

```diff
@@ -31,7 +31,7 @@ libraryDependencies ++= Seq(
   "com.gu" %% "identity-test-users" % "0.8",
   "com.google.guava" % "guava" % "29.0-jre",
   "io.lemonlabs" %% "scala-uri" % scalaUriVersion,
-  "com.gu.play-googleauth" %% "play-v27" % "2.1.0",
+  "com.gu.play-googleauth" %% "play-v27" % "2.2.2",
```

Comment:
existing, but that's a bit odd, why play-v27?

Reply:
oh I missed that! I'm surprised it even works